docs/gamejam.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+		<title>MakeCode Arcade Game Jam</title>
+		<meta name="twitter:card" content="summary_large_image">
+		<meta name="twitter:url" content="https://arcade.makecode.com/gamejam">
+		<meta name="twitter:title" content="MakeCode Arcade Heist Jam">
+		<meta name="twitter:site" content="@MsMakeCode">
+		<meta name="twitter:creator" content="@MsMakeCode">
+		<meta name="twitter:description" content="[desc here]">
+		<meta name="twitter:image" content="[img here]">
+		<link rel="stylesheet" href="https://pxt.azureedge.net/blob/2163189fd5e35c0981ed55318415582a7c9aeb12/doccdn/semantic.css" type="text/css">
+		<link rel="stylesheet" href="/static/gamejam/gamejam.css">	
+	</head>
+	
+<body>
+	<div class="hero">
+		<div class="banner"><img src="/static/gamejam/img/banner.png" /></div>
+	</div>
+	<div class="content">
+		<h1>MakeCode Arcade Game Jam</h1>
+		<div class="segment">
+			<p>
+				Welcome to the 3rd Official Microsoft MakeCode Game Jam! This is a fun 
+				competition where you can pit your game development skills against 
+				others to build a game using <a href="https://arcade.makecode.com">MakeCode Arcade</a>, 
+				a game engine for retro pixel-art games that run in the browser 
+				or on handheld game devices.
+			</p>
+			<div id="timer"></div>
+		</div>
+		<div class="segment">
+			<h2>Livestream Schedule</h2>
+			<p>Weekly sessions on game design and implementation from professional game developers!</p>
+			<div id="schedule"></div>
+			<div class="actions">
+				<a class="button" href="https://mixer.com/MakeCode" target="_blank" rel="noopener noreferrer">
+					Watch on Mixer
+				</a>
+			</div>
+		</div>
+		<div class="segment">
+			<div id="rules"></div>
+		</div>
+		<div class="segment">
+			<h2>Submit</h2>
+			<p>You can submit your game either through Microsoft Forms or itch.io.</p>
```

Comment:
Use `<href a=https://forms.office.com/>Microsoft Forms</a>` here? Hyperlink itch.io also?

Reply:
Let’s not leak user to random web sites.

Reply:
Yea, sorry. I didn't see the buttons. I went straight to the review page.